portal_backend/scheme_utils/trusted_aps_import.py

```diff
@@ -97,11 +115,60 @@ def __init__(self, all_APs_dict: dict, AP: str) -> None:
             self.contact_email = self.all_APs_dict[self.AP]["Issuer"]["ContactEMail"]
             self.logo_path = self.all_APs_dict[self.AP]["logo_path"]
             self.base_url = self.all_APs_dict[self.AP]["Issuer"].get("baseURL")
+            self.credentials = self.all_APs_dict[self.AP].get("credentials", {})
 
         except Exception as e:
             raise Exception(f"Error extracting fields from issuer: {e}")
 
 
+class CredentialFields:
+    def __init__(self, credential_dict: dict) -> None:
+        try:
+            spec = credential_dict.get("IssueSpecification", {})
+            self.credential_id = spec.get("CredentialID")
+            name = spec.get("Name", {})
+            self.name_en = name.get("en")
+            self.name_nl = name.get("nl")
+            shortname = spec.get("ShortName", {})
+            self.shortname_en = shortname.get("en")
+            self.shortname_nl = shortname.get("nl")
+            self.description_en = spec.get("Description", {}).get("en")
+            self.description_nl = spec.get("Description", {}).get("nl")
+            self.should_be_singleton = (
+                spec.get("ShouldBeSingleton", "").lower() == "true"
+            )
+            self.issue_url = None
+            issue_url = spec.get("IssueURL")
+            if isinstance(issue_url, dict):
+                self.issue_url = issue_url.get("en") or issue_url.get("nl")
+            else:
+                self.issue_url = issue_url
+
+            # Convert DeprecatedSince from UNIX Timestamp to date format
+            deprecated_raw = spec.get("DeprecatedSince")
+            if deprecated_raw:
+                try:
+                    if deprecated_raw.isdigit():  # Unix timestamp
+                        dt = datetime.utcfromtimestamp(int(deprecated_raw))
+                        self.deprecated_since = dt.date().isoformat()  # 'YYYY-MM-DD'
+                    else:
+                        self.deprecated_since = deprecated_raw  # assume valid date
+                except Exception as e:
+                    raise Exception(f"Invalid DeprecatedSince value: {e}")
+            else:
+                self.deprecated_since = None
+
+            attributes = spec.get("Attributes", {}).get("Attribute")
+            if attributes is None:
+                self.attributes = []
```

Comment:
Won't a fallback value do the same thing? It's used for everything else :)

```suggestion
            attributes = spec.get("Attributes", {}).get("Attribute", [])
```